manifests/params.pp

```diff
@@ -28,6 +28,7 @@
   $selinux_enabled                   = undef
   $socket_group                      = undef
   $labels                            = []
+  $daemon_environment_files          = []
```

Comment:
It doesn't look like this variable is used. The code is  little inconsistent at the moment with blank defaults. For the moment I'd just delete this line and leave the default in the init.pp/